dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.mpMetrics-2.2-monitor-1.0.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.mpMetrics-2.2-monitor-1.0
+visibility=private
+singleton=true
+IBM-Provision-Capability: \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.mpMetrics-2.2))", \
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.monitor-1.0)))"
+-bundles=com.ibm.ws.microprofile.metrics.2.0.monitor
+IBM-Install-Policy: when-satisfied
+kind=ga
```

Comment:
beta first please

Reply:
2.2 reverts back to 2.0, as the change in 2.1 was the problem, our team discussed that there will be no issues to go straight to GA.

Reply:
you have to go to beta until ALL feature approvals are obtained